tests/Extensions/Diagnostics.HealthChecks.UnitTests/HttpEndpointHealthCheckTests.cs

```diff
@@ -127,7 +127,7 @@ private async Task<(MockClient, HealthCheckResult)> RunHealthCheckAsync(HttpHeal
 
 			Mock<ICodePackageActivationContext> mock = new Mock<ICodePackageActivationContext>();
 			mock.Setup(c => c.GetEndpoint(parameters.EndpointName)).Returns(new System.Fabric.Description.EndpointResourceDescription());
-			ServiceContext context = MockStatelessServiceContextFactory.Create(mock.Object, "", new Uri("http://localhost"), Guid.Empty, 0);
+			ServiceContext context = MockStatelessServiceContextFactory.Create(mock.Object, "", new Uri("https://localhost"), Guid.Empty, 0);
```

Comment:
I realise you didn't add this, but it would be good to replace `""` with `string.Empty` as you're making this change given that the latter is more efficient

Reply:
Good point, replaced. 
Just in case if you are interested, I remember hearing something that JIT in netcore,  replaces calls to `""` with `String.Empty` , I was not able to find any exact mentions of it, but resulted JIT Asm looks quite simular in both cases https://sharplab.io/#v2:EYLgHgbALANALiAhgZwLYB8ACAmAjAWAChMBmAAhzIGEyBvIsxi8zKMgWQAoBKOhpgZlwBOTgGU4AJwCWAOwDmAOgCiqAA5wAntwDc/AYyGiARMd36yAXyKWgA==

Reply:
Thanks for the info. In older versions of .NET, there was definitely an issue as many called this out. I know there was a concerned effort in .NET Core to improve perf, so I guess this change was made as part of that. I still think making the change makes sense as my understanding is that we want this project to work in both .NET Framework and .NET Core